.github/workflows/e2e-nvidia-a10g-x4.yml

```diff
@@ -164,6 +164,19 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Post job results to Slack
+        if: ${{ always() }}
+        id: slack
+        uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
+        with:
+          # Slack channel id, channel name, or user id to post message.
+          # See also: https://api.slack.com/methods/chat.postMessage#channels
+          # You can pass in multiple channels to post to by providing a comma-delimited list of channel IDs.
+          channel-id: 'cli-e2e-results'
+          slack-message: "*e2e-nvidia-a10g-x4* job in *${{ github.repository }}* running on *${{ fromJSON(steps.check_pr.outputs.is_pr) && format('PR #{0}', steps.check_pr.outputs.pr_or_branch) || format('Branch {0}', steps.check_pr.outputs.pr_or_branch) }}* completed with result: *${{ job.status }}*"
```

Comment:
We're going to get these messages when someone runs on a PR?  Do we want that?

Reply:
Hmmm, I guess it does comment on the PR, we could have both but I see the argument for just doing the runs against `main` - I wouldn't mind having it to start personally and removing later if we find it redundant

Reply:
Works for me :+1: